lib/core/config.js

```diff
@@ -222,8 +232,29 @@ Config.prototype.loadContractsConfigFile = function() {
 
   let configFilePath = this._getFileOrOject(this.configDir, 'contracts', 'contracts');
 
-  const newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
-
+  let newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
+  if (newContractsConfig.gas != "auto") {
+      newContractsConfig.gas = utils.getWeiBalanceFromString(newContractsConfig.gas, web3);
+  }
```

Comment:
I think we could do sth. like:

```
newContractsConfig.gas = newContractsConfig.gas != 'auto' ?
  utils.getWeiBalanceFromString(...) :
  newContractsConfig.gas;
```

Well... as I look at this... it doesn't necessarily make it better.

But maybe we can at least make this magic `"auto"` string a constant?

Reply:
I think our eslint doesn't like multi-line ternaries. I also think that the current implementation is clearer too.